src/panels/lovelace/common/get-element-tag.ts

```diff
@@ -0,0 +1,7 @@
+const CUSTOM_TYPE_PREFIX = "custom:";
+
+export function getElementTag(type: string): string {
```

Comment:
This name is too generic, it's for cards, not just any element. Same with the file name.